src/middleware/packages/ldp/services/resource/actions/get.js

```diff
@@ -71,7 +77,16 @@ module.exports = {
           };
         }
 
-        return result;
+        if (!forceSemantic && result['@type'] === 'semapps:file') {
+          stream = fs.readFileSync(result['semapps:localpath']);
+          ctx.meta.$responseType = result['semapps:mimetype'];
+          ctx.meta.$responseHeaders = {
+            'Content-Disposition': `attachment; filename="${result['semapps:filename']}"`
+          };
```

Comment:
j'ai pas vraiment compris le probleme. J'ai vérifié qu'un navigateur affiche bien l'image si l'uri est mis en src. tu ne mettrait pas du tout de 'content-disposition'?

Reply:
`'Content-Disposition': `attachment;` déclenche un téléchargement au lieu d'afficher dans le navigateur.
> The first parameter in the HTTP context is either inline (default value, indicating it can be displayed inside the Web page, or as the Web page) or attachment (indicating it should be downloaded; most browsers presenting a 'Save as' dialog, prefilled with the value of the filename parameters if present).
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

Reply:
ok, je 'ai enlevé et tout continue à marcher correctement